### PR TITLE
Rename beamware to nerveshub

### PR DESCRIPTION
**Pull request by mobileoverlord**

### CODE REVIEW

File: config/prod.exs [hunk 1]
```diff
@@ -13,14 +13,14 @@ use Mix.Config
 # containing the digested version of static files. This
 # manifest is generated by the mix phx.digest task
 # which you typically run after static files are built.
-config :beamware, BeamwareWeb.Endpoint,
+config :nerveshub, NervesHubWeb.Endpoint,
   load_from_system_env: true,
-  url: [scheme: "https", host: "beamware.io", port: 80],
+  url: [scheme: "https", host: "nerveshub.io", port: 80],
```
Comment (fhunleth) [inline config/prod.exs]: nerves-hub.com

File: lib/nerveshub/accounts/email.ex [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+  use Phoenix.Swoosh, view: NervesHubWeb.EmailView, layout: {NervesHubWeb.LayoutView, :email}
 
-  alias Beamware.Accounts.{Invite, Tenant, User}
-  alias Beamware.Mailer
+  alias NervesHub.Accounts.{Invite, Tenant, User}
+  alias NervesHub.Mailer
   alias Swoosh.Email
 
-  @from {"Beamware", "no-reply@beamware.io"}
+  @from {"NervesHub", "no-reply@nerveshub.io"}
```
Comment (fhunleth) [inline lib/nerveshub/accounts/email.ex]: nerves-hub.com

File: lib/beamware/accounts/email.ex [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 
-  @from {"Beamware", "no-reply@beamware.io"}
+  @from {"NervesHub", "no-reply@nerves-hub.com"}
 
   def send(:invite, %Invite{} = invite, %Tenant{} = tenant) do
     %Email{}
     |> from(@from)
     |> to(invite.email)
-    |> subject("Welcome to Beamware!")
+    |> subject("Welcome to NervesHub!")
```
Comment (fhunleth) [inline lib/beamware/accounts/email.ex]: When we write it in English, it's "Nerves<space>Hub", right?

Comment (mobileoverlord) [reply lib/beamware/accounts/email.ex]: Good question. I guess so. Or we could do `Nerves-Hub` But I think that its okay to do `NervesHub`

**Pull request merged**